tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1287,7 +1347,7 @@ def _rostopic_cmd_type(argv):
 def _rostopic_cmd_hz(argv):
     args = argv[2:]
     from optparse import OptionParser
-    parser = OptionParser(usage="usage: %prog hz /topic", prog=NAME)
+    parser = OptionParser(usage="usage: %prog hz [options] /topic_0 [/topic_1 [[topic_2 [..]]]]", prog=NAME)
```

Comment:
I will replace the double `[[` with a single `[` when cherry-picking the changes.